src/theme/Admonition/index.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react'
```

Comment:
Have admonitions been removed from docusaurus?

Reply:
No. https://docusaurus.io/docs/markdown-features/admonitions. But the old configuration is no longer supported. The new way of configuring is apparently with Swizzling.